app/src/main/java/com/example/umlandowallet/Start.kt

```diff
@@ -63,48 +65,42 @@ fun start(
         (System.currentTimeMillis() * 1000).toInt()
     )
 
-    // Read Channel Monitor state from disk
-    // Initialize the hashmap where we'll store the `ChannelMonitor`s read from disk.
-    // This hashmap will later be given to the `ChannelManager` on initialization.
-    var channelMonitors = arrayOf<ByteArray>()
-    if (serializedChannelMonitors != "") {
-        Log.i(LDKTAG, "LDK: initiating channel monitors...")
-        val channelMonitorHexes = serializedChannelMonitors.split(",").toTypedArray()
-        val channelMonitorList = ArrayList<ByteArray>()
-        channelMonitorHexes.iterator().forEach {
-            val channelMonitorBytes = it.toByteArray()
-            channelMonitorList.add(channelMonitorBytes)
-        }
-        channelMonitors = channelMonitorList.toTypedArray()
-    }
-
     // This is going to be the fee policy for __incoming__ channels. they are set upfront globally:
     val userConfig = UserConfig.with_default()
     val newChannelConfig = ChannelConfig.with_default()
-    newChannelConfig.set_forwarding_fee_proportional_millionths(10000)
-    newChannelConfig.set_forwarding_fee_base_msat(1000)
-    userConfig.set_channel_config(newChannelConfig)
+    newChannelConfig._forwarding_fee_proportional_millionths = 10000
+    newChannelConfig._forwarding_fee_base_msat = 1000
+    userConfig._channel_config = newChannelConfig
 
 
-    val handshake = ChannelHandshakeConfig.with_default()
-    handshake.set_minimum_depth(1)
-    userConfig.set_channel_handshake_config(handshake)
+    val channelHandShakeConfig = ChannelHandshakeConfig.with_default()
+    channelHandShakeConfig._minimum_depth = 1
+    channelHandShakeConfig._announced_channel = false
+    userConfig._channel_handshake_config = channelHandShakeConfig
 
-    val newLim = ChannelHandshakeLimits.with_default()
-    newLim.set_force_announced_channel_preference(false)
-    userConfig.set_channel_handshake_limits(newLim)
+    val channelHandshakeLimits = ChannelHandshakeLimits.with_default()
+    channelHandshakeLimits._max_minimum_depth = 1
+    channelHandshakeLimits._force_announced_channel_preference = false
+    userConfig._channel_handshake_limits = channelHandshakeLimits
 
     val params = ProbabilisticScoringParameters.with_default()
-    val defaultScorer = ProbabilisticScorer.of(params, Global.router, logger).as_Score()
-    val scorer = MultiThreadedLockableScore.of(defaultScorer)
-    Global.scorer = scorer
+    val defaultScorer = ProbabilisticScorer.of(params, Global.router, logger)
+    val scoreRes = ProbabilisticScorer.read(
```

Comment:
I still need to dig into this probabilistic scorer... remind me to ask you about it next time we speak!